CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -1,5 +1,20 @@
 # Changelog 1.6
 
+## 1.6.1 - 2022-05-11
+
+### Added
+
+- Add [documentation](https://github.com/antrea-io/antrea/blob/main/docs/security.md#protecting-your-cluster-against-privilege-escalations) for the Antrea Agent RBAC permissions and how to restrict them using Gatekeeper/OPA. ([#3694](https://github.com/antrea-io/antrea/pull/3694),[@antoninbas])
+
+### Fixed
+
+- Clean up stale routes installed by AntreaProxy when ProxyAll is disabled. ([#3465](https://github.com/antrea-io/antrea/pull/3465), [@hongliangl])
+- Fix Services with port name fail to realize when trying to export/import them via Antrea Multi-cluster feature. ([#3561](https://github.com/antrea-io/antrea/pull/3561), [@luolanzone])
```

Comment:
That's not a correct sentence. I suggest:

> Fix export/import of Services with named ports when using the Antrea Multi-cluster feature.

Reply:
done. thanks!